plugins/search/src/components/SearchFilterNext/SearchFilterNext.tsx

```diff
@@ -0,0 +1,177 @@
+  label: {
+    textTransform: 'capitalize',
+  },
+});
+
+export type Component = {
+  className?: string;
+  name: string;
+  values?: string[];
+  defaultValue?: string[] | string | null;
+};
+
+export type Props = Component & {
+  component: (props: Component) => ReactElement;
+  debug?: boolean;
+};
+
+const CheckboxFilter = ({
+  className,
+  name,
+  defaultValue,
+  values = [],
+}: Component) => {
+  const classes = useStyles();
+  const { filters, setFilters } = useSearch();
+
+  useEffect(() => {
+    if (Array.isArray(defaultValue)) {
+      setFilters(prevFilters => ({
+        ...prevFilters,
+        [name]: defaultValue,
+      }));
+    }
+  }, [name, defaultValue, setFilters]);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const {
+      target: { value, checked },
+    } = e;
+
+    setFilters(prevFilters => {
+      const { [name]: filter, ...others } = prevFilters;
+      const rest = ((filter as string[]) || []).filter(i => i !== value);
+      const items = checked ? [...rest, value] : rest;
+      return items.length ? { ...others, [name]: items } : others;
+    });
+  };
+
+  return (
+    <FormControl
+      className={className}
+      fullWidth
+      data-testid="search-checkboxfilter-next"
+    >
+      <FormLabel className={classes.label}>{name}</FormLabel>
+      {values.map((value: string) => (
+        <FormControlLabel
+          key={value}
+          control={
+            <Checkbox
+              color="primary"
+              tabIndex={-1}
+              inputProps={{ 'aria-labelledby': value }}
+              value={value}
+              name={value}
+              onChange={handleChange}
+              checked={((filters[name] as string[]) ?? []).includes(value)}
+            />
+          }
+          label={value}
+        />
+      ))}
+    </FormControl>
+  );
+};
+
+const SelectFilter = ({
+  className,
+  name,
+  defaultValue,
+  values = [],
+}: Component) => {
+  const classes = useStyles();
+  const { filters, setFilters } = useSearch();
+
+  useEffect(() => {
+    if (typeof defaultValue === 'string') {
+      setFilters(prevFilters => ({
+        ...prevFilters,
+        [name]: defaultValue,
+      }));
+    }
+  }, [name, defaultValue, setFilters]);
+
+  const handleChange = (e: ChangeEvent<{ value: unknown }>) => {
+    const {
+      target: { value },
+    } = e;
+
+    setFilters(prevFilters => {
+      const { [name]: filter, ...others } = prevFilters;
+      return value ? { ...others, [name]: value as string } : others;
+    });
+  };
+
+  return (
+    <FormControl
+      className={className}
+      variant="filled"
+      fullWidth
+      data-testid="search-selectfilter-next"
+    >
+      <InputLabel className={classes.label} margin="dense">
+        {name}
+      </InputLabel>
+      <Select
+        variant="outlined"
+        value={filters[name] || ''}
+        onChange={handleChange}
+      >
+        <MenuItem value="">
+          <em>All</em>
```

Comment:
hm isn't there like `<ListItemText primary="All" />` or similar for this purpose in mui so we get the consistent styling

Reply:
There is!  But it turns out the style is actually no different than regular menu items.  ...Left as is for now.